language/de/acp/posting.php

```diff
@@ -91,6 +91,20 @@
 		'LOCAL_URL'		=> 'Eine lokale URL. Muss relativ zur Themenansicht angegeben werden. Protokoll und Domain darf nicht vorangestellt werden, da den Links „%s“ vorangestellt wird.',
 		'RELATIVE_URL'	=> 'Eine relative URL. Kann verwendet werden, um Teile einer URL zu prüfen. Achtung: auch eine vollständige URL ist eine gültige relative URL. Wenn relative URLs zur Adresse des Boards verwendet werden sollen, sollte der LOCAL_URL-Token verwendet werden.',
 		'COLOR'			=> 'Eine HTML-Farbe. Es kann entweder der hexadezimale Wert (z.&nbsp;B. <samp>#FF1234</samp>) oder ein <a href="http://www.w3.org/TR/CSS21/syndata.html#value-def-color">CSS-Farbwert</a> wie z.&nbsp;B. <samp>fuchsia</samp> oder <samp>InactiveBorder</samp> angegeben werden.',
+		'ALNUM'			=> 'Lateinische Buchstaben (A-Z) und Ziffern.',
+		'CHOICE'		=> 'Eine Auswahl von vorgegebenen Werten, z.B. <samp>{CHOICE=Pik,Herz,Karo,Kreuz}</samp>. Die Groß- und Kleinschreibung der Werte wird standardmäßig ignoriert. Die Groß- und Kleinschreibung kann beachtet werden, indem zusätzlich die Option <samp>caseSensitive</samp> angegeben wird: <samp>{CHOICE=Pik,Herz,Karo,Kreuz;caseSensitive}</samp>',
+		'FLOAT'			=> 'Ein Dezimalwert, z.B. <samp>0.5</samp>.',
+		'HASHMAP'		=> 'Verbindet Text mit seinem jeweiligen Ersatzwert in der Form <samp>{HASHMAP=Text1:Ersatzwert1,Text2:Ersatzwert2}</samp>. Groß- und Kleinschreibung wird beachtet. Unbekannte Werte werden standardmäßig beibehalten.',
+		'INT'			=> 'Eine Ganzzahl, z.B. <samp>2</samp>.',
+		'IP'			=> 'Eine gültige IPv4- oder IPv6-Addresse.',
+		'IPPORT'		=> 'Eine gültige IPv4- oder IPv6-Addresse mit Port.',
+		'IPV4'			=> 'Eine gültige IPv4-Addresse.',
+		'IPV6'			=> 'Eine gültige IPv6-Addresse.',
+		'MAP'			=> 'Verbindet Text mit seinem jeweiligen Ersatzwert in der Form <samp>{MAP=text1:ersatzwert1,text2:ersatzwert2}</samp>. Groß- und Kleinschreibung wird ignoriert. Unbekannte Werte werden standardmäßig beibehalten.',
+		'RANGE'			=> 'Akzeptiert eine Ganzzahl im angegebenen Bereich, z.B. <samp>{RANGE=-10,42}</samp>.',
+		'REGEXP'		=> 'Validiert den Wert mit dem angegebenen regulären Ausdruck, z.B. <samp>{REGEXP=/^foo\w+bar$/}</samp>.',
+		'TIMESTAMP'		=> 'Konvertiert einen Zeitstempel wie <samp>1h30m10s</samp> in eine Anzahl Sekunden. Akzeptiert auch eine Zahl.',
```

Comment:
"Konvertiert einen Zeitstempel wie <samp>1h30m10s</samp> in seine Anzahl in Sekunden."?

Reply:
Finde ich etwas seltsam. Wie wäre "[...] in die entsprechende Anzahl Sekunden."?

Reply:
"[...] in die entsprechende Anzahl Sekunden." finde ich gut

Reply:
Oder so: 
> 'Rechnet einen Zeitstempel wie <samp>1h30m10s</samp> in Sekunden um. Akzeptiert auch eine Zahl.',

Reply:
Hab's jetzt mal mit "entsprechende Anzahl Sekunden" ersetzt. Das "Rechnet ... um" passt mir irgendwie nicht in unseren bisherigen Sprachstil rein.